src/gradualizer.erl

```diff
@@ -25,12 +31,12 @@
 %% API functions
 
 %% @doc Type check a source or beam file
--spec type_check_file(file:filename()) -> ok | nok.
+-spec type_check_file(file:filename()) -> [{file:filename(), any()}].
```

Comment:
Is there no way of *not* returning the errors? If they are printed, they don't have to be returned, do they? The terminal will be flooded when running in an Erlang shell. (I usually do that.)

Reply:
Would you be ok with a wrapper,`gradualizer:checked_ok/1`?

```Erlang
checked_ok(Errors) -> length(Errors) =:= 0.
```

I'd prefer to not add an option since that makes for strange types and most likely a more complicated implementation.